### ISSUE TITLE
Release 267.4 introduced network configuration failures

### ISSUE THREAD

**Issue by williammartin** [cloudfoundry/bosh#2033]
When using `cf-deployment` to deploy Cloud Foundry on a bosh-lite director deployed via `bosh-deployment`, we are consistently hitting an error:

```
Task 35 | 14:50:11 | Preparing deployment: Preparing deployment (00:00:08)
                   L Error: Can't generate network settings without an IP
Task 35 | 14:50:19 | Error: Can't generate network settings without an IP
```

This has only started occurring since this [commit](https://capi.ci.cf-app.com/teams/main/pipelines/bosh-lite/jobs/create-bosh-lite/builds/594) to `bosh-deployment` that bumped the `bosh-release` from `266.7` to `267.4`. It consistently fails as soon as we attempt to use a director deployed via that commit.

This has broken pipelines for both Cloud Controller and the Services API components in Cloud Foundry. 

We can work around this for now by rolling back to an old version of `bosh-deployment`.

We may be able to provide reproduction steps locally in time.

**Comment by voelzmo** [comment]
Related thread on slack: https://cloudfoundry.slack.com/archives/C02HPPYQ2/p1534375069000100

**Comment by williammartin** [comment]
This was reproducible by following the standard `bosh-deployment` and `cf-deployment` instructions for a local bosh deployment from current master :(

https://bosh.io/docs/bosh-lite/
https://github.com/cloudfoundry/cf-deployment/blob/master/deployment-guide.md

**Comment by mfine30** [comment]
Should be fixed in [v267.5.0](https://github.com/cloudfoundry/bosh/releases/tag/v267.5.0)

**Comment by dpb587-pivotal** [comment]
Closing - v267.5.0 version is now available from [bosh-deployment](https://github.com/cloudfoundry/bosh-deployment).
